src/audio/volume/Kconfig

```diff
@@ -76,4 +76,25 @@ config COMP_GAIN
 	  This option enables gain to change volume. It works
 	  as peak volume without updating peak vol to host
 
+config VOLUME_HIFI4
+    bool
+    default y if "$(HIFI_VERSION)" = "4"
+    default n
+    help
+        This option used to enable HIFI4 optimized volume code for sof build
+
+config VOLUME_HIFI3
+    bool
+    default y if "$(HIFI_VERSION)" = "3"
+    default n
+    help
+        This option used to enable HIFI3 optimized volume code for sof build
+
+config VOLUME_GENERIC
+    bool
+    default y if "$(HIFI_VERSION)" = "generic" || "$(HIFI_VERSION)" = ""
```

Comment:
seems better if we initialize the HIFI_VERSION = "generic" in scripts/xtensa-build-zephyr.py